<launch>

    <node name="utw" pkg="cpp_uav" type="utw.py" output="screen" />
    <node name="torres_etal_2016" pkg="cpp_uav" type="torres_etal_2016" output="screen"/>

</launch>

<!-- vim: set et ft=xml fenc=utf-8 ff=unix sts=0 sw=4 ts=4 : -->
